### PR TITLE
List contacts sort last seen on

### PR DESCRIPTION
**Pull request by norkans7**
Requires mailroom v5.7.40

### CODE REVIEW

**Comment by norkans7** [comment]
Step 2 for https://github.com/rapidpro/rapidpro/issues/1366

**Comment by rowanseymour** [comment]
@norkans7 screenshot.. my only concern here is we're filling up that list view especially for orgs with lots of featured fields.. but then we do hide those automatically so maybe it's fine

**Comment by norkans7** [comment]
Contacts 2020-10-27 19-08-59
Contacts 2020-10-27 19-08-35

File: templates/contacts/contact_list.haml [hunk 1]
```diff
@@ -187,6 +187,25 @@
                               %div
                                 {{ field.label }}
                                 %span.sort.icon.icon-arrow-down-2.sort-desc
+                    %th.px-6.py-3.pt-4.border-b.border-gray-200.bg-gray-100.text-left.font-medium.uppercase.text-xs.tracking-wider
+                      - if object_list
+                        -if sort_field == 'last_seen_on'
+                          -if sort_direction == 'desc'
+                            %a{href:"?sort_on=last_seen_on{% if search %}&search={{ search|escape }}{% endif %}"}
```
Comment (rowanseymour) [inline templates/contacts/contact_list.haml:194]: `%a(href="`

File: templates/contacts/contact_list.haml [hunk 2]
```diff
@@ -187,6 +187,25 @@
                               %div
                                 {{ field.label }}
                                 %span.sort.icon.icon-arrow-down-2.sort-desc
+                    %th.px-6.py-3.pt-4.border-b.border-gray-200.bg-gray-100.text-left.font-medium.uppercase.text-xs.tracking-wider
+                      - if object_list
```
Comment (rowanseymour) [inline templates/contacts/contact_list.haml:191]: `-if`

**Comment by rowanseymour** [comment]
@norkans7 thus the value of screenshots!

File: templates/contacts/contact_list.haml [hunk 3]
```diff
@@ -227,6 +246,12 @@
                           %td.field
                             {{ object|contact_field:field.key }}
 
+                      %td.value-received.field_received
+                        .whitespace-no-wrap
+                          -if object.last_seen_on
+                            {% format_datetime object.last_seen_on %}
+                          - else
```
Comment (rowanseymour) [inline templates/contacts/contact_list.haml:253]: `-else`

**Review by rowanseymour — APPROVED** [review]

**Pull request merged**